central/main.go

```diff
@@ -651,19 +653,27 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 	})
 
 	scannerDefinitionsRoute := "/api/extensions/scannerdefinitions"
+	// Only grant compression to well-known content types. It should capture files
+	// worth of compression in definition's bundle. Ignore all other types (e.g.,
+	// `.zip` for the bundle itself).
+	definitionsFileGzipHandler, _ := gziphandler.GzipHandlerWithOpts(gziphandler.ContentTypes([]string{
```

Comment:
Also, we should do a `utils.CrashOnError` here instead of ignoring the error

Reply:
It's best to leave it outside the handler so it can be used (that includes tests) with or without the compression logic, which is decoupled from the handler itself.